src/ex3/Tema3.java

```diff
@@ -0,0 +1,32 @@
+package ex3;

```

Comment:
this package should reside in `com.itschoolsession4.homework` (like you did with `ex1` and `ex2`), not here independent.